docs/engines/flink/flink-cdc-ingestion.md

```diff
@@ -0,0 +1,547 @@
+    Schema.UnresolvedPrimaryKey userPrimaryKey = userSchema.getPrimaryKey().orElseThrow(() -> new RuntimeException("table user required pk "));
+    ResolvedSchema userResolvedSchema = new ResolvedSchema(userTableCols, Collections.emptyList(), UniqueConstraint.primaryKey(
+      userPrimaryKey.getConstraintName(), userPrimaryKey.getColumnNames()));
+    ResolvedCatalogTable userTable = new ResolvedCatalogTable(
+      CatalogTable.of(userSchema, "", Collections.emptyList(), new HashMap<>()), userResolvedSchema);
+    pathAndTable.add(Tuple2.of(new ObjectPath("test_db", "user"), userTable));
+
+    // build table "product"
+    Schema productSchema = Schema.newBuilder()
+      .column("productId", DataTypes.INT().notNull())
+      .column("price", DataTypes.DECIMAL(12, 6))
+      .column("saleCount", DataTypes.INT())
+      .primaryKey("productId")
+      .build();
+    List<Column> productTableCols = Stream.of(
+      Column.physical("productId", DataTypes.INT().notNull()),
+      Column.physical("price", DataTypes.DECIMAL(12, 6)),
+      Column.physical("saleCount", DataTypes.INT())).collect(Collectors.toList());
+    Schema.UnresolvedPrimaryKey productPrimaryKey = productSchema.getPrimaryKey().orElseThrow(() -> new RuntimeException("table product required pk "));
+    ResolvedSchema productResolvedSchema = new ResolvedSchema(productTableCols, Collections.emptyList(), UniqueConstraint.primaryKey(
+      productPrimaryKey.getConstraintName(), productPrimaryKey.getColumnNames()));
+    ResolvedCatalogTable productTable = new ResolvedCatalogTable(
+      CatalogTable.of(productSchema, "", Collections.emptyList(), new HashMap<>()), productResolvedSchema);
+    pathAndTable.add(Tuple2.of(new ObjectPath("test_db", "product"), productTable));
+    return pathAndTable;
+  }
+
+  private static Map<String, RowDataDebeziumDeserializeSchema> getDebeziumDeserializeSchemas(
+    final List<Tuple2<ObjectPath, ResolvedCatalogTable>> pathAndTable) {
+    return pathAndTable.stream()
+      .collect(toMap(e -> e.f0.toString(), e -> RowDataDebeziumDeserializeSchema.newBuilder()
+        .setPhysicalRowType(
+          (RowType) e.f1.getResolvedSchema().toPhysicalRowDataType().getLogicalType())
+        .setUserDefinedConverterFactory(MySqlDeserializationConverterFactory.instance())
+        .setMetadataConverters(
+          new MetadataConverter[] {TABLE_NAME.getConverter(), DATABASE_NAME.getConverter()})
+        .setResultTypeInfo(TypeInformation.of(RowData.class)).build()));
+  }
+}
+```
+
+### Mixed-Iceberg format
+The following example will show how to write CDC data from multiple MySQL tables into the corresponding Mixed-Iceberg table.
+
+**Requirements**
+
+Please add [Flink Connector MySQL CDC](https://mvnrepository.com/artifact/com.ververica/flink-connector-mysql-cdc/2.3.0) and [Amoro](https://mvnrepository.com/artifact/com.netease.arctic/arctic-flink-runtime-1.14/0.4.1) dependencies to your Maven project's pom.xml file.
```

Comment:
`https://mvnrepository.com/artifact/com.netease.arctic/arctic-flink-runtime-1.14/0.4.1`
Is this link correct?